nginx-proxy/maintenance.conf

```diff
@@ -13,11 +13,13 @@ server {
   }
 
   location / {
+    if ($content_type ~* "text/html") {
```

Comment:
I'm trying to imagine the scenario where this would happen. Would we be getting a non-html page while the site is down?

Reply:
When something is trying to use the API

Reply:
Ah I see, and 503 is Service Unavailable. Cool 👍